recipes/nnetsauce/LICENSE.txt

```diff
@@ -0,0 +1,32 @@
+The Clear BSD License
```

Comment:
If you are building from a github release, the license file will already be included so you can remove this file and also change `license_file` in the `meta.yaml` to just `license_file: LICENSE` since that's the naming convention in the upstream repository.